Rarime/Code/Modules/ScanPassport/ViewModels/PassportViewModel.swift

```diff
@@ -78,7 +78,40 @@ class PassportViewModel: ObservableObject {
         
         AppUserDefaults.shared.lastMRZKey = value
     }
-
+    
+    func getPassportInfo(_ document: Passport?, _ proof: ZkProof) async throws -> PassportInfo {
```

Comment:
pass a non-nullable passport here and get rid of name mismatch `document` & `Passport`